packages/angular/src/generators/ngrx/files/classes-syntax/__directory__/__fileName__.facade.spec.ts__tmpl__

```diff
@@ -57,54 +57,42 @@ describe('<%= className %>Facade', () => {
     /**
      * The initially generated facade::loadAll() returns empty array
      */
-    it('loadAll() should return empty list with loaded == true', async (done) => {
-      try {
-        let list = await readFirst(facade.all<%= className %>$);
-        let isLoaded = await readFirst(facade.loaded$);
+    it('loadAll() should return empty list with loaded == true', async () => {
```

Comment:
I'd be interested in hearing your thoughts on the following.

Are you aware of the AAA Testing Pattern?
http://dontcodetired.com/blog/post/Arrange-Act-Assert-Comments-in-Tests 
(In that article I disagree that the comments should be removed but that's a personal opinion)

Essentially, looking at the tests we generate, we do:

Arrange, Assert, Act, Assert

I feel like this adds complexity to the test.

If we refactored the tests slightly it might read a little better e.g.

```ts
// ARRANGE
const listBeforeLoaded = await readFirst();
const isLoadedBefore = await readFirst()

// ACT
facade.loadAll()

// ASSERT
const list = await readFirst()
const isLoaded = await readFirst()

expect(isLoadedBefore).toBe(false);
expect(isLoaded).toBe(true)
```

Something along those lines.

I'm not saying you have to do it, but I'd be interested in hearing your thoughts on it!

Reply:
Hey @Coly010. Yeah, those test would certainly read better. That would be a good improvement, in this PR I was just removing deprecated syntax, changeset looks large due to fixing indentation after removing a try catch that was not needed.